### DISCUSSION TITLE
Missing content unless entry is saved/published

### DISCUSSION THREAD

**Discussion by kylef-fotf** [statamic/cms#8313] [Troubleshooting] [answered]
I have over 1k entries imported, all set to published, and CP shows they are published already.

When I try to grab them in a template, it doesn't list them, even if I specify status as published, and a limit.

The code I'm trying to use is simply:
 ```
<ul>
{{ collection:articles }}
<li><a href="{{url}}">{{title}}</a></li>
{{ /collection:articles }}
</ul>
```
When I save/publish an entry (that is already published) it will then appear in the index properly.

However, I don't want to hit save/publish on every entry we have, and I don't want to have to repeat it for every import we do.

Does anyone know why this might be happening or how to troubleshoot it?

**Comment by duncanmcclean** [comment]
Have you tried clearing your cache? `php artisan cache:clear`

**Reply by kylef-fotf** [reply]
Yes, but it doesn't change anything.

**Reply by duncanmcclean** [reply]
Cool - just wanted to check that first.

Are you able to share the file contents of one of the entries that's published in the CP but not showing as such in the front-end?

**Reply by kylef-fotf** [reply]
We are using eloquent-driver for entries.

The first row is an article that will not display in the front end until I hit save & publish.
The second row is an article that is displaying, because I hit save & publish after importing.

<table>
<tr>
<th>id</th>
<th>site</th>
<th>origin_id</th>
<th>published</th>
<th>status</th>
<th>slug</th>
<th>uri</th>
<th>date</th>
<th>order</th>
<th>collection</th>
<th>data</th>
<th>created_at</th>
<th>updated_at</th>
</tr>
<tr>
<td>6462</td>
<td>default</td>
<td>null</td>
<td>1</td>
<td>published</td>
<td>not-visible-in-front-end</td>
<td>content/not-visible-in-front-end</td>
<td>null</td>
<td>null</td>
<td>articles</td>
<td>{"body":"<p>Example HTML data wrapped in json</p>", "title":"Example title","id":"11151","categories":["cat-1","cat-2","cat-3"]}</td> 
<td>2019-01-03 19:56:37</td>
<td>2019-05-28 19:54:07</td>
</tr>
<tr>
<td>7162</td>
<td>default</td>
<td>null</td>
<td>1</td>
<td>published</td>
<td>visible-in-front-end</td>
<td>content/visible-in-front-end</td>
<td>2019-05-28 00:00:00</td>
<td>null</td>
<td>articles</td>
<td>{"body":"<p>Example HTML data wrapped in json</p>", "title":"Example title","id":"8748"}</td> 
<td>2017-09-05 22:06:51</td>
<td>2023-06-15 21:53:32</td>
</tr>

**Reply by ryanmitchell** [reply, marked as answer]
Is it a dated collection? If so the date being null is probably the reason

**Reply by kylef-fotf** [reply]
I attached an image as well, to show what it looks like in cp. You can see it looks like everything is published.

![Untitled-1](https://github.com/statamic/cms/assets/104449622/ae44a7f9-720d-4849-be11-bf1645b62adc)

**Reply by ryanmitchell** [reply]
Yes but you can see the date is null in the column so it’s defaulting to todays date. Set a last date where it’s null and they will likely appear. Though I can’t be sure as you didn’t answer my question.

**Reply by kylef-fotf** [reply]
> Is it a dated collection? If so the date being null is probably the reason

Yes it's dated, and that is what's stopping it. Thanks for helping me find that.
What's confusing is it's displaying a date on everything, and if I were to open any article up, they're pre-populated with an old date (nothing is dated today, unless I create a brand new entry from cp). I'm guessing it's taking the value of `created_at`.

**Comment by ryanmitchell** [comment]
When you say you imported them, what method did you use? 
You may find `php please stache:warm` resolves the issue for you.

**Reply by kylef-fotf** [reply]
We used a data import addon from the addon explorer. It imports via CSV.
I just tried running `php please stache:warm` and it did not resolve it.
